### ISSUE TITLE
Pipe operator inside quotation expression leads to wrong indentation

### ISSUE THREAD

**Issue by danyx23** [fsprojects/fantomas#256]
The following expression:
```fsharp
test <@ results.GetAllResults() |> set = expected_outcome @>
```
Gets reformatted to this, which fails to compile (the equals sign should be aligned with set):
```fsharp
test <@ results.GetAllResults()
        |> set
        = expected_outcome @>
```

Does it make more sense to report these things in the form of failing unit tests? If so, this here fails:
```fsharp
[<Test>]
let ``Quotation expression with pipe and comparision``() =
    formatSourceString false """
    test <@ results.GetAllResults() |> set = expected_outcome @>""" config
    |> prepend newline
    |> should equal """
test <@ results.GetAllResults()
        |> set
           = expected_outcome @>
"""
```

**Comment by danyx23** [comment]
Might be a duplicate of #248

**Comment by nojaf** [comment]
Failing unit test is the best way to report this. Interesting case.

**Comment by jindraivanek** [comment]
Fixed by #294.
